README.md

```diff
@@ -1,591 +1,24 @@
-# OpenZeppelin Nile ⛵
+# OpenZeppelin | Nile ⛵
```

Comment:
I think the simplification looks very good, but maybe too minimal. we can keep the succinctness but i would at least add:

1. a 101 quickstart to get going immediately. clicking the docs link should not be an adoption barrier
2. one or two example commands being executed. this could be very well part of (1)
3. an overview of the API. it can be very short and high level, but to give a quick and more concrete idea than the overview

Reply:
I've made a refactor to the README adding installation steps, and improving the Documentation entry (I think it looks better now). For the rest of the things, I'm not in favor of adding them into the README of a mid/big-sized project. 

I'm more in favor of something similar to what hardhat, truffle, starknet-devnet, protostar, and others are doing.

Our quick start guide is complex (deploying an account is a requirement before even deploying contracts, and that requires some steps that are better presented in the doc site with TIPs, NOTEs, etc...). And the API/CLI Reference is quite big, and I think it would be confusing to present something partial in the README.

IMO the project looks cleaner in general with a minimalistic presentation in the README and a well-organized / well-presented doc site. 

With the documentation entry in the README, users can quickly go to the Quickstart, or the Plugin list, or the Guides, in one click after reaching the project. Not requiring a Ctrl+F and not being exhausted with a lot of information they are probably not looking for at the moment.

<img width="466" alt="Screenshot 2022-12-07 at 18 00 15" src="https://user-images.githubusercontent.com/26070224/206242902-519e6b4c-acf4-4665-b5fe-8c7ded05fab8.png">

Reply:
IMO is also a good practice to avoid having to touch the README often by including API References or a complex Quickstart, when we need to keep this information up-to-date in the doc site. We will be struggling some times with the presentation if we want to have a "good looking" README.

Reply:
not sure we're talking about the same, but don't see anything confusing nor "complex" about having one of two snippets showing `nile compile` and `nile call`, to name two. you actually added something very similar in the quickstart with `init` and `compile`. same idea.